examples/agentInteractionsDemo/app.js

```diff
@@ -207,9 +209,22 @@ new Vue({
                                 .then((conv) => {
                                     let newConv = conv;
                                     newConv.customer = conv.participants.find((part) => {
-                                        return part.purpose === "customer";
-                                    });
-                                    this.conversationsData.conversations.push(newConv);
+                                        return part.purpose === "customer" || part.purpose === "external";
+                                    }) || { name: "Unknown" };
+
+                                    if (newConv.customer.externalContactId){
+                                        console.log("externalContact", newConv)
+                                        externalContactsApi.getExternalcontactsContact(newConv.customer.externalContactId)
+                                            .then((externalContact) => {
+                                                newConv.customer.name = `${externalContact.firstName} ${externalContact.lastName}`;
+                                                this.conversationsData.conversations.push(newConv);
+                                            })
+                                    } else{
+                                        if(!newConv.customer.name){
+                                            newConv.customer.name = "Unknown";
```

Comment:
Added this here because there's an edge case where an agent sends an outbounding email to someone that is not an external contact, the conversation object will not have the name field.